examples/src/main/java/io/dapr/examples/exception/README.md

```diff
@@ -111,17 +111,3 @@ Once running, the OutputBindingExample should print the output as follows:
 ...
 
 ```
-
-### Cleanup
-
-To stop the app run (or press `CTRL+C`):
-
-<!-- STEP
-name: Cleanup
--->
-
-```bash
-dapr stop --app-id exception_example
```

Comment:
Is this removed due to the related change in CLI to exit with error when app id is not found?
https://github.com/dapr/cli/commit/7b69e1a6c66fa58471b57e555bbfca3429e11dae
@halspang 
cc @artursouza

Reply:
Yes. With that change, a lot of the examples will finish and close everything successfully. When we attempted to close it we'd get error messages, hence the removal.